torch/_inductor/lowering.py

```diff
@@ -5132,6 +5132,97 @@ def all_to_all_single(
             )
         )
 
+    _c10d_functional = torch.ops._c10d_functional
+
```

Comment:
Just noting we're about to land Broadcast here as well, so there may be some merge conflicts to handle -https://github.com/pytorch/pytorch/pull/112668

Let me know if I can help.